docs/event-dispatcher.md

```diff
@@ -152,83 +217,88 @@ Example:
 
-- StackStx
-```json
-{
-  "stack_stx": {
-    "burn_block_height": 10,
-    "burn_header_hash": "1010101010101010101010101010101010101010101010101010101010101010",
-    "num_cycles": 10,
-    "reward_addr": "16Jswqk47s9PUcyCc88MMVwzgvHPvtEpf",
-    "sender": {
-      "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
-      "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
-      "address_version": 26
-    },
-    "stacked_ustx": 10,
-    "burn_txid": "0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a",
-    "vtxindex": 10
-  }
-}
-```
+#### Example JSON values for `burnchain_op` field:
+*   **TransferStx:**
+    ```json
+    {
+      "transfer_stx": {
+        "burn_block_height": 10,
+        "burn_header_hash": "1410131010105914101010101013704010101010221010101010101010101010",
+        "memo": "0x000102030405",
+        "recipient": {
+          "address": "SP24ZBZ8ZE6F48JE9G3F3HRTG9FK7E2H6K2QZ3Q1K",
+          "address_hash_bytes": "0x89f5fd1f719e4449c980de38e3504be6770a2698",
+          "address_version": 22
+        },
+        "sender": {
+          "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
+          "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
+          "address_version": 26
+        },
+        "transfered_ustx": 10,
+        "burn_txid": "85aa2106186723f3c4f1d8bb58e3a02746ca9be1be9f4be0c6557079e1f660e6",
+        "vtxindex": 10
+      }
+    }
+    ```
 
-- DelegateStx
-```json
-{
-  "delegate_stx": {
-    "burn_block_height": 10,
-    "burn_header_hash": "1010101010101010101010101010101010101010101010101010101010101010",
-    "delegate_to": {
-      "address": "SP24ZBZ8ZE6F48JE9G3F3HRTG9FK7E2H6K2QZ3Q1K",
-      "address_hash_bytes": "0x89f5fd1f719e4449c980de38e3504be6770a2698",
-      "address_version": 22
-    },
-    "delegated_ustx": 10,
-    "sender": {
-      "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
-      "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
-      "address_version": 26
-    },
-    "reward_addr": [10, "16Jswqk47s9PUcyCc88MMVwzgvHPvtEpf"],
-    "burn_txid": "0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a",
-    "until_burn_height": null,
-    "vtxindex": 10
-  }
-}
-```
+*   **StackStx:**
+    ```json
+    {
+      "stack_stx": {
+        "burn_block_height": 10,
+        "burn_header_hash": "1010101010101010101010101010101010101010101010101010101010101010",
+        "num_cycles": 10,
+        "reward_addr": "16Jswqk47s9PUcyCc88MMVwzgvHPvtEpf",
+        "sender": {
+          "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
+          "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
+          "address_version": 26
+        },
+        "stacked_ustx": 10,
+        "burn_txid": "0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a",
+        "vtxindex": 10
+      }
+    }
+    ```
+
+*   **DelegateStx:**
```

Comment:
would it be worthwhile to add an example for stacks-extend here?

i'm also curious about the naming pattern used, i.e. `StackStx` - is that part of the payload, or possibly we should use the function name from the pox contract, `stack-stx` https://github.com/stacks-network/stacks-core/blob/master/stackslib/src/chainstate/stacks/boot/pox-4.clar#L571